aerolyzer/horizon.py

```diff
@@ -1,12 +1,45 @@
-class horizon:
-	
-	def __init__(self):
-		self.message = "Hello World"
-	
+import sys
+import cv2
+import os
+import numpy as np
+from matplotlib import pyplot as plt
 
-	def _hello(self,i):
-		x=0
-		while x<i:
-			print self.message
-			x = x + 1
-		return 0
+def test():
+	print "hi"
+def _testAnalyze(self, directory):
+	included_extensions = ['jpg','JPG']
+	files = [fn for fn in os.listdir(directory) if any(fn.endswith(ext) for ext in included_extensions)]
+	for image in files:
+		img_noblur = cv2.imread(str(directory)+image)
+		img_noblur = cv2.cvtColor(img_noblur, cv2.COLOR_BGR2GRAY)
+		self.getHorizon(str(directory), img_noblur,image)
+
+def getHorizon(self, directory, img_noblur,image):
+	img = cv2.blur(img_noblur, (10,10))
+	img = cv2.equalizeHist(img)
+	img = cv2.equalizeHist(img)
+	img = cv2.cvtColor(img, cv2.COLOR_GRAY2BGR)
+	edges = cv2.Canny(img,150,200) #magic numbers (min & max tolerance)
+	plt.subplot(121),plt.imshow(img,cmap = 'gray')
+	plt.title('Original Image'), plt.xticks([]), plt.yticks([])
+	plt.subplot(122),plt.imshow(edges,cmap = 'gray')
+	plt.title('Edge Image'), plt.xticks([]), plt.yticks([])
+	plt.show()
```

Comment:
remove print statements that are used for debugging. If the function needs to return a string, please adjust accordingly.